_ingest-pipelines/processors/kv.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: KV
+parent: Ingest processors
+nav_order: 200
+redirect_from:
+   - /api-reference/ingest-apis/processors/lowercase/
+---
+
+# KV processor
+
+The `kv` processor automatically extracts specific event fields or messages that are in a`key=value` format. This structured format organizes your data by grouping it together based on keys and values. It's helpful for analyzing, visualizing, and using data such as user behavior analytics, performance optimizations, or security investigations. 
```

Comment:
Last sentence: I suggest just "It's helpful in use cases such as user behavior analytics...". Otherwise, "...and using data, such as user behavior analytics, performance optimization, or security investigation data."